src/ucp/api/ucp.h

```diff
@@ -629,6 +631,54 @@ void ucp_worker_release_address(ucp_worker_h worker, ucp_address_t *address);
  */
 void ucp_worker_progress(ucp_worker_h worker);
 
+/**
+ * @ingroup UCP_WORKER
+ * @brief Obtain an event file descriptor for interrupt notification.
+ *
+ * This routine returns a valid file descriptor for polling functions.
+ * The file descriptor will get signaled on any event on one of the underlying
```

Comment:
this needs to be described in UCP terminology - what the event means for UCP-layer operations